README.md

```diff
@@ -1,23 +1,19 @@
-RAMSES
+<!-- RAMSES -->
```

Comment:
This is just nitpicks but it made me curious:
This looks like a HTML comment?  Did you intend this and is it for some internal purpose?   I would guess that it isn't strictly an approved format for MarkDown.

(I agree however with the result which is that it is not rendered by GitHub and the title is I suppose not needed there.)

Reply:
Yes, I intended for it to not be displayed. 
It's not a bad idea to have the top of the readme labeled with name of the project, which I assumed is what you wanted. It did, however, look a little odd with no formatting, so I just made it hidden.
If you don't mind not having it labeled at all, you can delete it.

Reply:
Sure. It’s not up to me, I was just reviewing out of interest.